### DISCUSSION TITLE
Re-encoding a big FLAC8 library into FLAC0 without hassle

### DISCUSSION THREAD

**Discussion by githubgavin1337** [enzo1982/freac#652] [Help & Support] [answered]
I have always used the highest compression level of 8 for my FLAC files for all my lossless audio library needs, but I have recently come to the conclusion that the marginal file size decrease over even the lowest compression level of 0 is just not worth the extra CPU overhead it takes to read back those more heavily comporessed files, so I would like to re-encode my entire FLAC8 library into FLAC0 with the least amount of hassle possible. Is there a simple way to do it in fre:ac so that it would just automatically overwrite the old files without changing anything else like file names, locations, or metadata etc so that after the encoding hurdle I could just hop right back onto my favourite media player using the same old playlist and start listening again without having to do any extra work with file management? Sorry for the noobish question but I tried googling about this topic and couldn't find a tutorial on how to do it, and I'm a bit iffy about going about it through trial and error in case I mess my library up.

**Comment by cleanyoursockets** [comment, marked as answer]
I suggest you create a test directory with a couple of folders in it and make a test run. I use Freac a lot for this purpose and it works well. You will not experience any hassle if your metadata are in order.

**Comment by githubgavin1337** [comment]
Ok I was fiddling around a bit and found that in "Options -> General settings setup -> fre:ac -> Output files" there are checkboxes for both using the input folder as the output folder as well as overwriting the source files whenever possible, and those seemed to do the trick.
